arm-sql/compositeSql.json

```diff
@@ -0,0 +1,23 @@
+{
+  "info": {
+    "title": "SqlManagementClient",
+    "description": "Composite Swagger for Azure Sql Database Client"
+  },
+  "documents": [
+    //"./2015-05-01/swagger/jobaccount.json",
```

Comment:
@nathannfan - We still see the commented line in the composite document. Can you please make sure to remove this line?
> //"./2015-05-01/swagger/jobaccount.json",